components/card/demo/basic.md

```diff
@@ -12,25 +12,49 @@ title:
 ## en-US
 
 A basic card containing a title, content and an extra corner content.
+Supports three different sizes: `default`, `middle` and `small`.
 
 ````jsx
 import { Card } from 'antd';
 
 ReactDOM.render(
-  <Card
-    title="Card title"
-    extra={<a href="#">More</a>}
-    style={{ width: 300 }}
-  >
-    <p>Card content</p>
-    <p>Card content</p>
-    <p>Card content</p>
-  </Card>,
+  <div>
+    <Card
+      title="Default size card"
+      extra={<a href="#">More</a>}
+      style={{ width: 300 }}
+    >
+      <p>Card content</p>
+      <p>Card content</p>
+      <p>Card content</p>
+    </Card>
+    <Card
+      size="middle"
```

Comment:
order of cards demo: `default -> small -> large`

Reply:
default is actually the largest size, so it goes `large -> medium -> small` at the moment.

Reply:
you can look here: https://ant.design/components/list/ how demo shows the order

Reply:
Ah I see, I based it on the Table instead of List. The problem with List is that their default size is actually the medium one. For Card, the default size is already the largest size. So I don't think I can copy LIst exactly